### ISSUE TITLE
Midterm Review

### ISSUE THREAD

**Issue by calliebusch** [medhavi27/Hot-Spotify-Tracks#10]
This group is using features about songs on Spotify like tempo, key, and artist to predict a song's popularity. The dataset consists of many numeric variables, which will translate well into regression, and some categorical variables, like artist, which will add a lot of complexity to the model if used. (Think about how many artists are on Spotify/ in the dataset of 10,000 songs.) They have already begun building a set of models, but plan on using transformations to build them out. 

Pros: 
Many covariates seem well distributed for building linear models (close to normal). 
Using MAE is a great way to mitigate extremely popular songs' potential to add bias to the model. 
It's smart that you are being selective on which models are being run, so you don't risk finding a good model out of luck. 

Improvements: 
I would love to know how many artists are in your dataset. If there are 8000, this could lead to overfitting. (Artist maps directly to popularity, leaving out other features). Especially when this is one hot encoded, you may have as many features as examples. 
Have you considered using other loss functions (Hinge Loss), alongside MAE? 
It would have been great to see visualizations showing covariate distributions compared to the outcome variable or one another. This could help prove/disprove multicollinearity, which can affect which variables should be used.
